front-end/src/util/blockToTime.ts

```diff
@@ -0,0 +1,62 @@
+// Copyright 2019-2020 @paritytech/polkassembly authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+import BN from 'bn.js';
+import { chainProperties } from 'src/global/networkConstants';
+
+import getNetwork from './getNetwork';
+
+type Time = [number, number, number, number];
+
+const HRS = 60 * 60;
+const DAY = HRS * 24;
+
+function addTime (a: Time, b: Time): Time {
+	return [a[0] + b[0], a[1] + b[1], a[2] + b[2], a[3] + b[3]];
+}
+
+function extractTime (value?: number): Time {
```

Comment:
I didn't have a deep look at this but it sounds convoluted. SO showed me a nice way for seconds -> Dhms. It's here: https://stackoverflow.com/a/52387803/3086912

Reply:
This is so much better, thanks for the link :)